kmk/kmktime.py

```diff
@@ -1,23 +1,22 @@
+import supervisor
+
+import const
 import time
 
+_TICKS_PERIOD = const(1 << 29)
+_TICKS_MAX = const(_TICKS_PERIOD - 1)
+_TICKS_HALFPERIOD = const(_TICKS_PERIOD // 2)
+
 
 def sleep_ms(ms):
     return time.sleep(ms / 1000)
 
 
 def ticks_ms():
-    '''Has .25s granularity, but is cheap'''
-    return time.monotonic() * 1000
+    return supervisor.ticks_ms() * 1000
```

Comment:
The `*1000` seems wrong, since the units are already ms.  Do you want to eliminate this function and instead supply it with `from supervisor import ticks_ms` so no wrapper is needed?

Reply:
Good call.